include/ignition/gui/qml/GzColor.qml

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (C) 2022 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+import QtQuick 2.9
+import QtQuick.Controls 2.1
+import QtQuick.Dialogs 1.0
+import QtQuick.Layouts 1.3
+
+
+Item {
+  id: gzColorRoot
+  
+  implicitWidth: 40
+  implicitHeight: 40
+
+  property double r: 255
```

Comment:
Let's go ahead and update colors to be between `0 - 1.0`, that way we don't have to divide by `255` everywhere. Also, can you add a comment above `Item {` that states rgba values are between `0-1` to help implementors?